crates/nu_plugin_stress_internals/src/main.rs

```diff
@@ -114,6 +114,8 @@ pub fn main() -> Result<(), Box<dyn Error>> {
             Err(err) => {
                 if err.is_eof() {
                     break;
+                } else if err.is_io() {
+                    std::process::exit(1);
```

Comment:
Is there an implication for the design of the engine side we can draw from the failure that prompted you to add this?

Reply:
Oh - alternatively, since this happened on Windows, I wonder if maybe the pipe just isn't readable anymore after the program exits, unlike Unix sockets. If that's the case then maybe what we need to do is just do a real flush on Drop so that we wait for the other side to have read everything before we let it close.